src/App.tsx

```diff
@@ -15,36 +15,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const enum SortType {
+  SORT_ALPHABETICALLY = 'alphabetically',
+  SORT_BY_LENGTH = 'length',
+  default = '',
+}
```

Comment:
<img width="1048" alt="Screenshot 2024-02-26 at 17 16 08" src="https://github.com/mate-academy/react_list-of-goods-hooks/assets/14590796/b9e7f1bd-fadd-49f0-be68-9fb7a3c168c7">